indicaAi/spec/models/local_rating_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe LocalRating, type: :model do
+  it 'Testing if value is integer - Should return true' do
```

Comment:
Começar a descrição do teste com letra minúscula, é o padrão da comunidade do Rails. E de preferência mais para o estilo `should return true is value is integer`